### PR TITLE
add new modern era

### PR DESCRIPTION
**Pull request by Cyrildab**

### CODE REVIEW

**Review by pagouillart — APPROVED** [review]
Evil genius !

File: frontend/src/components/LevelPage/ModernLevelPage/Modern.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      item: { itemName, itemSrc, itemClass, itemClassButton },
+      imgPopUpClass,
+    };
+  }
+  const napoleon = getModernData(1, "napoleon", "napoleonPopUp");
+  const rousseau = getModernData(
+    2,
+    "rousseau",
+    "rousseauPopUp",
+    "oeuf de quetzalcoatlus",
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ModernLevelPage/Modern.jsx]: Really bad copy paste

File: frontend/src/components/LevelPage/ModernLevelPage/Modern.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  const victorHugo = getModernData(4, "victorHugo", "victorHugoPopUp");
+  const louisXVI = getModernData(5, "louisXVI", "louisXVIPopUp");
+
+  return (
+    <>
+      <main className="modernTheme">
+        <GameMenu />
+
+        <TargetButton
+          dino={napoleon.data}
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ModernLevelPage/Modern.jsx]: Dino ? Are you sure ?

File: frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+.napoleonPopUp {
+  width: 7vw;
+  margin: 1rem;
+}
+
+.rousseau {
+  width: 6vw;
+  position: absolute;
+  top: 75%;
+  left: 92%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss]: Better with a right

File: frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
+@import "../../../variables.scss";
+
+.napoleon {
+  width: 6vw;
+  position: absolute;
+  top: 80%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss]: Better with a bottom

File: frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+.rousseauPopUp {
+  width: 7vw;
+  margin: 1rem;
+}
+
+.louisXVI {
+  width: 6vw;
+  position: absolute;
+  top: 72%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss]: Better with a bottom

File: frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+.louisXVIPopUp {
+  width: 7vw;
+  margin: 1rem;
+}
+
+.victorHugo {
+  width: 7vw;
+  position: absolute;
+  top: 75%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss]: Better with a bottom

File: frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  left: 62%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss]: Better with a right

File: frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+.victorHugoPopUp {
+  width: 7vw;
+  margin: 1rem;
+}
+
+.marieAntoinette {
+  width: 5vw;
+  position: absolute;
+  top: 51.2%;
+  left: 78%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss]: Better with a right

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/LevelPage/ModernLevelPage/Modern.jsx (hunk 1 above)
Comment (Cyrildab) [reply frontend/src/components/LevelPage/ModernLevelPage/Modern.jsx]: done

File: frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss (hunk 3 above)
Comment (Cyrildab) [reply frontend/src/components/LevelPage/TargetButton/TargetButtonModern.scss]: done

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**